src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -196,7 +196,7 @@ public SimpleIngestManager build() {
   // logger object for this class
   private static final Logger LOGGER = LoggerFactory.getLogger(SimpleIngestManager.class);
   // HTTP Client that we use for sending requests to the service
-  private HttpClient httpClient;
+  private CloseableHttpClient httpClient;
```

Comment:
Do we need to close the client somewhere?

Reply:
I was actually thinking of closing the client when `SimpleIngestManager` is closed, but then since httpClient cant remain static instance. Because atleast in KC, we call close on simplingestmaneger which closes the BG service of keyRenewal. 

If I add httpClient.close(), I will also have to wipe off httpClient from JVM since it is static?

I can do `httpClient = null` may be?

Reply:
But then again defeats the purpose of static instance. I would prefer going without close() on closeable instance.